dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/EditDataset.ts

```diff
@@ -24,6 +24,7 @@ describeIf(
       cy.fixture("CompanyInformationWithLksgPreparedFixtures").then(function (jsonContent) {
         const preparedFixtures = jsonContent as Array<FixtureData<LksgData>>;
         preparedFixture = getPreparedFixture("lksg-all-fields", preparedFixtures);
+        cy.pause();
```

Comment:
Please add a comment that indicates why this pause is added?

Reply:
yes, that's how it works. 
the css class is created by coookiebot.com
(If i wanted to do this with v-if i would need to ask for consent anyways and contact cookiebot for that)

Reply:
also pushed